<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="KRUPA" failures="2" tests="4" name="PetTest" time="16.077" errors="0" timestamp="2025-10-16T15:15:51 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="2.788"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="9.127">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testGetUserByName(UserTests.java:68)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="2.212"/>
  <testcase classname="api.test.UserTests" name="testDeleteUserByName" time="1.051">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testDeleteUserByName(UserTests.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- PetTest -->
